app/public/cantusdata/models/chant.py

```diff
@@ -5,9 +5,8 @@ class Chant(models.Model):
     """
     A Chant belongs to a image page (or a chant can appear
     on multiple pages?)
```

Comment:
at some point, it might be nice to have a definitive statement here, rather than a question :P

Reply:
yes, i would also like to know how we will solve this problem 😂😂

Reply:
Like, in practice, I know a chant can span multiple pages.

Here, could we say something along the lines of, "a Chant belongs to an image page. In physical manuscripts, the last chant on a page often continues onto the next page. Cantus Database associates each chant with the page on which it begins, and Cantus Ultimus follows this convention [or substitute whatever CU actually does]"? A statement like this would communicate some of the nuance of the situation and how CU handles this situation, as compared to the current statement which communicates something along the lines of "it's complicated and weird, and darned if I know what to do about it!".